semgrep-core/tests/lua/assignment.lua

```diff
@@ -0,0 +1,19 @@
+-- ERROR: match
+foo = foo
+
+foo = {}
+
+-- ERROR: match
+foo = foo
+
+qux = foo:bar() == foo:baz()
+
+qux = foo:bar() == foo:bar()
+
+local a, b, c = 1, 2, 3
+
+-- ERROR: match
+a, b, c = a
+
+-- ERROR: match
```

Comment:
Again, I don't think it should match.
This should be transformed into
Assign (a, Tuple(a,b,c))

Reply:
Same as above.